dashboard/test/factories/user_school_infos.rb

```diff
@@ -0,0 +1,19 @@
+# == Schema Information
```

Comment:
if you don't need a `:user_school_info` factory, you can probably delete this file for now. also, we have almost all of our factories in one file right now: https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/test/factories/factories.rb

not ideal, but just a heads up that when/if you do need this factory, you might want to add it there while we decide a better structure for our factories